modules/ssh-agent-using.adoc

```diff
@@ -54,51 +54,54 @@ endif::[]
 ifeval::["{context}" == "installing-ibm-z"]
 :ibm-z:
 endif::[]
+ifeval::["{context}" == "installing-rhv-default"]
+:rhv:
+endif::[]
+ifeval::["{context}" == "installing-rhv-customizations"]
+:rhv:
+endif::[]
 
 [id="ssh-agent-using_{context}"]
 = Generating an SSH private key and adding it to the agent
 
-If you want to perform installation debugging or disaster recovery on your cluster, you must provide an SSH key to both your `ssh-agent` and to the installation program.
+Generate a SSH key pair and add it to your `ssh-agent`. Later, you will provide the SSH public key to the installation program.
+
 [NOTE]
 ====
-In a production environment, you require disaster recovery and debugging.
+* Using SSH keys is essential for debugging and performing disaster recovery on your cluster.
+* For production environments, disaster recovery and debugging are required activities.
 ====
 
 ifdef::ibm-z[]
 [IMPORTANT]
 ====
-Do not skip this procedure in production environments where disaster recovery and debugging is required.
+Do not skip this procedure for production environments.
 ====
 endif::[]
 
-You can use this key to SSH into the master nodes as the user `core`. When you
-deploy the cluster, the key is added to the `core` user's
-`~/.ssh/authorized_keys` list.
+You can use this key to SSH into the master nodes as the user `core`. When you deploy the cluster, the key is added to the `core` user's `~/.ssh/authorized_keys` list.
 
-ifndef::osp,ibm-z[]
+ifndef::osp,ibm-z,rhv[]
 [NOTE]
 ====
-You must use a local key, not one that you configured with platform-specific
-approaches such as
-link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[AWS key pairs].
+You must use a local key, not one you configured with platform-specific approaches such as link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[AWS key pairs].
 ====
 endif::[]
 
 .Procedure
 
-. If you do not have an SSH key that is configured for password-less authentication
-on your computer, create one.
-For example, on a computer that uses a Linux operating system, run the
+IMPORTANT: If you create a new SSH key pair, avoid overwriting existing SSH keys.
```

Comment:
This note makes more sense attached to the first step.

Reply:
In the IBM Style Guide, the "Attention notices" topic says to "place the attention notice _before_ the step in which the risk exists." In this case, what would be the best location?
When I do this:
```
.Procedure

[IMPORTANT]
====
If you create a new SSH key pair, avoid overwriting existing SSH keys.
====

. If you do not have an SSH key that is configured for password-less authentication
```
![...makes-this](https://user-images.githubusercontent.com/350907/80325892-01861580-8805-11ea-97ab-ccdb6b49df54.png)